{
    "url": "https://api.github.com/repos/spack/spack/issues/11867",
    "repository_url": "https://api.github.com/repos/spack/spack",
    "labels_url": "https://api.github.com/repos/spack/spack/issues/11867/labels{/name}",
    "comments_url": "https://api.github.com/repos/spack/spack/issues/11867/comments",
    "events_url": "https://api.github.com/repos/spack/spack/issues/11867/events",
    "html_url": "https://github.com/spack/spack/pull/11867",
    "id": 461493837,
    "node_id": "MDExOlB1bGxSZXF1ZXN0MjkyNDA1OTU4",
    "number": 11867,
    "title": "Fixed a Python 2.6 parsing error",
    "user": {
        "login": "alalazo",
        "id": 4199709,
        "node_id": "MDQ6VXNlcjQxOTk3MDk=",
        "avatar_url": "https://avatars.githubusercontent.com/u/4199709?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/alalazo",
        "html_url": "https://github.com/alalazo",
        "followers_url": "https://api.github.com/users/alalazo/followers",
        "following_url": "https://api.github.com/users/alalazo/following{/other_user}",
        "gists_url": "https://api.github.com/users/alalazo/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/alalazo/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/alalazo/subscriptions",
        "organizations_url": "https://api.github.com/users/alalazo/orgs",
        "repos_url": "https://api.github.com/users/alalazo/repos",
        "events_url": "https://api.github.com/users/alalazo/events{/privacy}",
        "received_events_url": "https://api.github.com/users/alalazo/received_events",
        "type": "User",
        "site_admin": false
    },
    "labels": [
        {
            "id": 73908754,
            "node_id": "MDU6TGFiZWw3MzkwODc1NA==",
            "url": "https://api.github.com/repos/spack/spack/labels/bug",
            "name": "bug",
            "color": "fc2929",
            "default": true,
            "description": null
        },
        {
            "id": 456341797,
            "node_id": "MDU6TGFiZWw0NTYzNDE3OTc=",
            "url": "https://api.github.com/repos/spack/spack/labels/tests",
            "name": "tests",
            "color": "b60205",
            "default": false,
            "description": "General test capability(ies)"
        }
    ],
    "state": "closed",
    "locked": false,
    "assignee": null,
    "assignees": [],
    "milestone": null,
    "comments": 7,
    "created_at": "2019-06-27T12:14:28Z",
    "updated_at": "2019-08-11T20:29:12Z",
    "closed_at": "2019-08-11T20:00:38Z",
    "author_association": "MEMBER",
    "active_lock_reason": null,
    "pull_request": {
        "url": "https://api.github.com/repos/spack/spack/pulls/11867",
        "html_url": "https://github.com/spack/spack/pull/11867",
        "diff_url": "https://github.com/spack/spack/pull/11867.diff",
        "patch_url": "https://github.com/spack/spack/pull/11867.patch"
    },
    "body": "fixes #12268 \r\n\r\nApparently `shlex.split` can't deal with unicode encoded characters in Python 2.6. The solution is to convert to `str` before calling the function.",
    "performed_via_github_app": null
}